fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -280,16 +263,20 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
                 }
             }
 
+            val keyStore: KeyStore
```

Comment:
can you please explain, what is the goal to create local variable instead reinit the property one?